### PR TITLE
Product listing

### PR DESCRIPTION
**Pull request by govardhanreddyc79**
- Added Product listing page

image

### CODE REVIEW

File: src/pages/products/products.jsx [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+
+export const Products = () => {
+    const [ products, setProducts] = useState([])
+
+    useEffect(() => {
+        (async function () {
+            try{
+                let productItems = await axios.get("/api/products");
+                setProducts(productItems.data.products)
+            }catch (err) {
```
Comment (arpit01923) [inline src/pages/products/products.jsx:14]: add some error either console.log or use toast

**Review by arpit01923 — APPROVED** [review]
- clean UI
- good folder structure
- one suggestion add good PR description

File: src/pages/products/products.jsx (hunk 1 above)
Comment (govardhanreddyc79) [reply src/pages/products/products.jsx:14]: sure

File: src/backend/db/products.js [hunk 2]
```diff
@@ -8,23 +9,68 @@ import { v4 as uuid } from "uuid";
...
-    price: "5000",
-    categoryName: "non-fiction",
+    category: "Bats",
+    productName:"MRF Bat",
+    imageSource:bat1,
+    imageAlt:"MRF",
+    price:1800,
+    rating:4,
+    quantity:0,
+    link:"/ProductList"
```
Comment (SanketDhabarde) [inline src/backend/db/products.js]: Why add this links in DB

**Review by SanketDhabarde — APPROVED** [review]
Liked 
- the folder structure of the images
- You are using barrel export whenever necessary
- Destructuring the values instead of repeating
Good work 👌

File: src/backend/db/products.js (hunk 2 above)
Comment (govardhanreddyc79) [reply src/backend/db/products.js]: removed

**Pull request merged**